### ISSUE TITLE
Fall Damage

### ISSUE THREAD

**Issue by raedman90** [oscar-broman/samp-weapon-config#237]
Hello, how do I activate fall damage?

**Comment by NexiusTailer** [comment]
Its enabled by default

**Comment by raedman90** [comment]
Does not work, players do not drown or fall from great heights

**Comment by ADRFranklin** [comment]
Then there is a problem with your script, you are either hooking the callbacks yourself or using another library that is doing so, or you have stuff in `OnPlayerDamage` that is preventing it happening

**Comment by adib-yg** [comment]
I have the same problem.
I just added the weapon-config to a gamemode and players don't take damage from falling from heights, it was fine before I added it.

Then I started debugging.
`OnPlayerTakeDamage` and `OnPlayerGiveDamage` callbacks are not called in the include when the player falls from a height.
I think the data is sent by the client is overwritten by pawn.raknet.

I've tried hard to find the problem in include but can't find anything.
@oscar-broman please fix this.

**Comment by Hidend** [comment]
Yes it does seem to be realted to Pawn.Raknet, it does also happen only to certain clients, will investigate further but it does indeed happen on my end

**Comment by NexiusTailer** [comment]
> it does also happen only to certain clients

If all these client are android implementations, then it's probably their problem and a subject to fix on their side.
